### PR TITLE
add note about Office requirement

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5632

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md [hunk 1]
```diff
@@ -130,6 +130,9 @@ You can also use [Advanced hunting](advanced-hunting-query-language.md) to query
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md]:
```suggestion
> [!NOTE]
```

File: windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md [hunk 2]
```diff
@@ -130,6 +130,9 @@ You can also use [Advanced hunting](advanced-hunting-query-language.md) to query
...
 >The connection to the test machines is done using RDP. Make sure that your firewall settings allow RDP connections.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md]:
```suggestion
> The connection to the test machines is done using RDP. Make sure that your firewall settings allow RDP connections.
```

File: windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md [hunk 3]
```diff
@@ -130,6 +130,9 @@ You can also use [Advanced hunting](advanced-hunting-query-language.md) to query
...
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md]:
```suggestion
> [!NOTE]
```

File: windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md [hunk 4]
```diff
@@ -130,6 +130,9 @@ You can also use [Advanced hunting](advanced-hunting-query-language.md) to query
...
+>Microsoft Office needs to be installed on the test machines for all the simulations to work.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/evaluation-lab.md]:
```suggestion
> Microsoft Office needs to be installed on the test machines for all the simulations to work.
```

**Comment by mypil** [comment]
@mjcaparas - PR has been copy edited and is ready for final review. This is based on issue #5632

Thank you.

cc: @kenwith

**Comment by mjcaparas** [comment]
This statement is wrong:  _Microsoft Office needs to be installed on the test machines for all the simulations to work._

MS Office is already installed on the VMs.

**Comment by illfated** [comment]
Hmm... if that is the case, this PR is unneeded and the reported issue in ticket #5632 is incorrect, then.

**Comment by VLG17** [comment]
> This statement is wrong: _Microsoft Office needs to be installed on the test machines for all the simulations to work._
> 
> MS Office is already installed on the VMs.

@mjcaparas Thank you for confirming. I have asked the user to provide more details of what he's seeing during his tests.

**Comment by illfated** [comment]
Fair enough. At least you were able to make a couple of whitespace adjustments. Thank you.

**Pull request merged**